fabric-api-base/src/main/resources/fabric.mod.json

```diff
@@ -5,5 +5,8 @@
   "license": "Apache-2.0",
   "depends": {
     "fabricloader": ">=0.4.0"
+  },
+  "custom": {
+    "fabric-networking.shouldHandshake": false
```

Comment:
When shouldHandshake is disabled I assume no infomation about that mod is sent to the server.

Would there ever be a need or a want for a server to know about client side mods? I can see it being useful for the server to know if a client mod is installed and if sepecial handing should be abled for that player?

Reply:
Ah, so the client still sends the mod over, but just doesnt validate it?

Reply:
Yeah it sends a response of every single modcontainer within the client's loader.